optics-vl/optics-vl.cabal

```diff
@@ -31,7 +32,7 @@ library
 
   build-depends: base                   >= 4.9        && <5
                , indexed-profunctors    >= 0.1        && <0.2
-               , optics-core            >= 0.2        && <0.3
+               , optics-core            >= 0.2        && <0.4
```

Comment:
Should this be >0.3?

Reply:
No. I intend it as revision, so the `0.2` have to be allowed too (as it was). And there's no reason not to.

The changes since `v0.2` tag in `optics-vl` package are:

```diff
diff --git a/optics-vl/optics-vl.cabal b/optics-vl/optics-vl.cabal
index 9a90393..496fd79 100644
--- a/optics-vl/optics-vl.cabal
+++ b/optics-vl/optics-vl.cabal
@@ -1,12 +1,13 @@
 name:          optics-vl
 version:       0.2
+x-revision:    1
 license:       BSD3
 license-file:  LICENSE
 build-type:    Simple
 maintainer:    optics@well-typed.com
 author:        Andrzej Rybczak
 cabal-version: 1.24
-tested-with:   ghc ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.5 || ==8.8.1, GHCJS ==8.4
+tested-with:   ghc ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.5 || ==8.8.3 || ==8.10.1, GHCJS ==8.4
 synopsis:      Utilities for compatibility with van Laarhoven optics
 category:      Data, Optics, Lenses
 description:
@@ -31,7 +32,7 @@ library
 
   build-depends: base                   >= 4.9        && <5
                , indexed-profunctors    >= 0.1        && <0.2
-               , optics-core            >= 0.2        && <0.3
+               , optics-core            >= 0.2        && <0.4
                , profunctors            >= 5.0        && <6.0
 
   ghc-options: -Wall
```

Thus I don't think we need to make a release. Just a revision allowing `optics-core-0.3`.